src/mahomaps/screens/SettingsScreen.java

```diff
@@ -29,7 +29,7 @@ public class SettingsScreen extends Form implements CommandListener {
 	private ChoiceGroup download = new ChoiceGroup(MahoMapsApp.text[54], Choice.MULTIPLE,
 			new String[] { MahoMapsApp.text[17] }, null);
 	private ChoiceGroup map = new ChoiceGroup(MahoMapsApp.text[153], Choice.EXCLUSIVE,
-			new String[] { MahoMapsApp.text[55], MahoMapsApp.text[154], MahoMapsApp.text[155] }, null);
+			new String[] { MahoMapsApp.text[55],  MahoMapsApp.text[167], MahoMapsApp.text[154], MahoMapsApp.text[155] }, null);
```

Comment:
Map settings from previous versions will not be loaded correctly after that change, e. g satellite will be changed to dark, so I think there should be separate setting for scheme color

Reply:
I'm a bit dumb on this, so I probably can't make a separate parameter (yet) ;)